grafana/manifests/deis-monitor-grafana-deployment.yaml

```diff
@@ -1,28 +1,32 @@
-apiVersion: v1
-kind: ReplicationController
+apiVersion: extensions/v1beta1
+kind: Deployment
 metadata:
   name: deis-monitor-grafana
   namespace: deis
   labels:
     heritage: deis
 spec:
   replicas: 1
+  strategy:
+    rollingUpdate:
+      maxSurge: 1
+      maxUnavailable: 0
+    type: RollingUpdate
   selector:
-    app: deis-monitor-grafana
+    matchLabels:
+      app: deis-monitor-grafana
   template:
     metadata:
       labels:
         app: deis-monitor-grafana
     spec:
       containers:
       - name: deis-monitor-grafana
-        image: quay.io/deis/grafana:canary
+        image: quay.io/deis/grafana:v2.5.0
```

Comment:
oh I mean "can you change this back to :canary"? Right now it's pointing at v2.5.0.